helm/dscp-identity-service/values.yaml

```diff
@@ -5,6 +5,7 @@ config:
   dbName: dscp
   dbPort: 5432
   enableLivenessProbe: true
+  selfAddress: YYrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
```

Comment:
What is this default value?

Reply:
Just some random address

Reply:
It doesn't look valid. I thought all these identites started with a `5`? Anyway it should probably match the code default of `5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty`

Reply:
it worked when I tested locally, updated to what you have suggested, same as in devDefault